src/cow-react/modules/swap/helpers/getSwapButtonState.ts

```diff
@@ -27,6 +28,7 @@ export enum SwapButtonState {
   RegularSwap = 'RegularSwap',
   SwapWithWrappedToken = 'SwapWithWrappedToken',
   EthFlowSwap = 'EthFlowSwap',
+  EthFlowSwapBuyUnsupported = 'EthFlowSwapBuyUnsupported',
```

Comment:
heheh, incredible the amount of flavours we have for the same button :)

Reply:
In the newest version, this change is not required.
Take a look and tell me which one makes more sense

Reply:
I think it's cool when we can see the complete list of states, it's much easier to test having the list :)

Reply:
Sure, I can keep this for clarity and remove the input error

Reply:
> I think it's cool when we can see the complete list of states, it's much easier to test having the list :)

100%

Reply:
No longer relevant as we are going a different route